packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -128,7 +126,7 @@ export class FacetSearch {
   }
 
   private get clearButton() {
-    if (this.component.facetSearchQuery === '') {
+    if (this.facetSearchState.query === '') {
```

Comment:
Since we check this many time in the code, it could be a good idea to add a method 
```
private get isEmptyQuery() {}
```